src/cohort-builder.resource.ts

```diff
@@ -21,3 +21,28 @@ export const search = async (searchParams: SearchParams) => {
   );
   return searchResults;
 };
+
+/**
+ * @returns Locations
+ */
+export const fetchLocations = async () => {
+  const locationsResp: FetchResponse<{
+    results: LocationResponse[];
+  }> = await openmrsFetch("/ws/rest/v1/location", {
```

Comment:
Please use SWR instead of asynchronous network requests

Reply:
@vasharma05 I'm also not aware of `SWR`. this might be a stupid question; 😁 
what is the advantage of using it here?

Reply:
https://swr.vercel.app/ @jayasanka-sack
<img width="879" alt="Screenshot 2022-07-15 at 12 02 05 PM" src="https://user-images.githubusercontent.com/43912578/179165079-5fa1a4a5-0b89-4feb-b919-712456897bb8.png">